tests/fast-integration/eventing-test/README.md

```diff
@@ -0,0 +1,35 @@
+# Eventing Test
+
+This test covers end-to-end flow for Kyma eventing.
```

Comment:
```suggestion

## Overview 

This test covers the end-to-end flow for Eventing.
```
Just a small change according to the [template](https://github.com/kyma-project/community/blob/main/templates/repository-template/README.md) + we usually refer to the area as Eventing instead of Kyma Eventing. Thank you for updating the readme 🙂